client/me/security-2fa-backup-codes-list/index.jsx

```diff
@@ -61,6 +63,15 @@ module.exports = React.createClass( {
 		return true;
 	},
 
+	saveCodesToFile: function( event ) {
+		event.preventDefault();
+
+		const { user_login } = this.props.userSettings.settings;
```

Comment:
I don't see this prop being used elsewhere on the page, I realize that although unlikely, do we need to guard against `userSettings` being undefined?

Another route would be to use [lib\user](https://github.com/Automattic/wp-calypso/tree/master/client/lib/user) to get at the login name.